pkg/operation/care/seed_health_test.go

```diff
@@ -387,7 +388,7 @@ func progressingManagedResource(name string) *resourcesv1alpha1.ManagedResource
 
 func managedResource(name string, conditions []gardencorev1beta1.Condition) *resourcesv1alpha1.ManagedResource {
 	namespace := v1beta1constants.GardenNamespace
-	if name == "istio" || name == "istio-system" {
+	if name == "istio" || name == "istio-system" || strings.HasSuffix(name, "-istio") {
```

Comment:
Which managed resource has the `-istio` suffix? I am a bit surprised by this.

Reply:
It can happen, when you [add a prefix to the istio managed resource](https://github.com/gardener/gardener/blob/17dfefaffed6c5e125e35b6614c8dcad801839f1/pkg/component/istio/istiod.go#L321). 
However, `-istio` is not entirely correct, it should be `istio`.

Reply:
I guess that the first condition is than redundant, i.e. `strings.HasSuffix(name, "istio")` also yields `true` for `name == "istio"`.
Do you want skip the first condition to make it more concise?